packages/host/app/components/matrix/chat-sidebar.gts

```diff
@@ -14,6 +14,9 @@ import type MatrixService from '../../services/matrix-service';
 interface Args {
   Args: {
     onClose: () => void;
+    onCommand: (command: any) => void;
```

Comment:
It's the passing up and down of these functions, onCommand onPreviewCommand and onCancelPreviewCommand

Reply:
this kind of pattern is fine for invoking functions that reside "higher up" in the component tree--in ember we call these callbacks "actions". I guess the deeper question is: where is the correct place for these actions to live.